ska_helpers/version.py

```diff
@@ -22,7 +22,7 @@
     warnings.filterwarnings(
         "ignore", message=r"Module \w+ was already imported", category=UserWarning
     )
-    from pkg_resources import DistributionNotFound, get_distribution
+    from importlib import resources, metadata
```

Comment:
This seems fine, though unlike pkg_resources, we're already importing importlib at the top of the file.  Does this need to be in this catch_warnings block?

Reply:
Good point. I'm fighting two instincts, (1) cut any code that doesn't seem necessary (and re-test) and (2) let it go and approve and move on. I'm honestly not sure, but otherwise this PR looks good and we should get it merged one way or the other.

Reply:
These two sub-modules are not imported by the import statement at the top of the file, so they have to be explicitly imported. This goes to say that the fact that importlib was already imported says nothing about whether the warning would be issued here

I don't know whether importing without the warnings filter would trigger the warning. I do not remember if there were special cases when this was triggered, or if a simple call to testr is enough to check.

Reply:
I just removed the filter warning in the **master** branch, and I did not get this warning, so I do not know in which circumstances this warning was issued to begin with.

Reply:
Seems fine to leave this as is.